src/types/User.ts

```diff
@@ -14,6 +14,8 @@ export type VideoState = {
   removedParticipantUids?: string[];
 };
 
+export type MyPersonalizedSchedule = Record<string, string[]>;
```

Comment:
please, cover that into Partial<> because the current implementation implies that for every string there is a string array, which is not true

Reply:
What is the 'key' `string` meant to represent here out of curiosity? And what is the 'value' `string[]`?

Reply:
@0xdevalias the string key is VenueId, string[] is an array of eventIds

Reply:
Makes sense.

[NB] I wonder if we could make better use of type aliases or similar to make these sorts of types more ‘self documenting’? Though maybe that would just become more confusing.

Reply:
fixed and added comment-description: https://github.com/sparkletown/sparkle/pull/1302/commits/f9a5a03fd4b27519a1be12954168717088125af5

please, let me know if it looks unclear.

Reply:
Comment seems good enough